### ISSUE TITLE
Unreadable error message in telegram

### ISSUE THREAD

**Issue by PeterVoronov** [simatec/ioBroker.backitup#652]
**Describe the bug**  
The example of message
```
****Y****o****u****r**** ****b****a****c****k****u****p**** ****w****a****s**** ****n****o****t**** ****c****o****m****p****l****e****t****e****l****y**** ****c****r****e****a****t****e****d****.**** ****P****l****e****a****s****e**** ****c****h****e****c****k**** ****t****h****e**** ****e****r****r****o****r****s****!****!****
****
****c****i****f****s****:**** ****E****r****r****o****r****:**** ****E****N****O****E****N****T****:**** ****n****o**** ****s****u****c****h**** ****f****i****l****e**** ****o****r**** ****d****i****r****e****c****t****o****r****y****,**** ****o****p****e****n**** ****'****/****o****p****t****/****i****o****b****r****o****k****e****r****/****b****a****c****k****u****p****s****/****i****o****b****r****o****k****e****r****_****2****0****2****2****_****0****3****_****2****3****-****1****0****_****2****2****_****5****1****_****i****o****b****_****b****a****c****k****u****p****i****o****b****r****o****k****e****r****.****t****a****r****.****g****z****'****
```

**To Reproduce**  
1. Setup simple backup, only for iobroker.
2. Make NAS/Copy option enabled.
3. On page NAS/Copy select Copy method. Without any other options. Define any local folder for copy.
4. Enable Telegram reporting.
5. Make a backup - everything is Ok.
6. To simulate any error, change ownership and rights of /opt/iobroker/backups, for example root:root, 700.
7. Make a backup - will receive the same type of message.

**Expected behavior**  
Readable message in telegram, without `****`.

**Versions:**  
 - Adapter version: 2.3.5
 - JS-Controller version: 4.0.21
 - Node version: 14.19.1
 - Operating system: Debian 11

**Comment by PeterVoronov** [comment]
[Pull request](https://github.com/simatec/ioBroker.backitup/pull/653) proposed.

**Comment by simatec** [comment]
Fix in Version 2.4.0

**Comment by PeterVoronov** [comment]
> Fix in Version 2.4.0

thanks
